docs/api/hooks.md

```diff
@@ -404,17 +402,14 @@ import { bindActionCreators } from 'redux'
 import { useDispatch } from 'react-redux'
 import { useMemo } from 'react'
 
-export function useActions(actions, deps) {
+export function useActions(actions, deps = []) {
```

Comment:
according to: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters#evaluated_at_call_time
default parameters are evaluated at call time.  So this code creates new array with each call.

Such approach is not recommended for core libraries (unnecessary memory waste).

Reply:
I don't see how this makes any difference, because typically you're allocating a new array when you call the hook anyway: `useSomeHookWithDeps(arg1, [])`.

Reply:
Allocating a new hook-deps array in your business code is a consensus between code readability and usability.

But here we are dealing with a library in which we are not sure how it is used. For example, someone can write 100 such hooks in the code. And this additionally creates 100 new arrays.

Therefore, it is much better to create one static array in a singleton:

```
const defaultDeps = [] // we only create a new array once

export function useActions (actions, deps = defaultDeps) {
```

All we want Redux to be the most efficient framework, right? So we should not remove any microoptimizations, that were here in this code (even if they were not added intentionally ;) )